src/components/Comments/CommentsBar.tsx

```diff
@@ -0,0 +1,45 @@
+import { Close as CloseIcon } from '@mui/icons-material';
+import { Box, Drawer, Stack, Typography } from '@mui/material';
+import { IconButton } from '../IconButton';
+import { useCommentsContext } from './CommentsBarContext';
+import { CommentsThreadList } from './CommentsThreadList';
+
+interface CommentsBarProps {
+  resourceId?: string;
+}
+
+export const CommentsBar = ({ resourceId }: CommentsBarProps) => {
+  const { isCommentsBarOpen, toggleCommentsBar } = useCommentsContext();
+  const open = isCommentsBarOpen && !!resourceId;
+  const width = 350;
+
+  return (
+    <Drawer
```

Comment:
Currently this drawer can cover main layout content. It would be better to treat this is something in the layout so the main content is pushed to the left. Happy to brainstorm on ways we could do this when you get here.